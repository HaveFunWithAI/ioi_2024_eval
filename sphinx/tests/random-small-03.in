50 49
19 16 26 11 46 36 4 14 33 26 34 15 0 7 16 35 24 19 21 26 35 12 34 35 16 20 17 19 41 14 3 19 36 22 34 13 11 47 17 12 47 11 37 46 12 21 25 4 33 37
10 15
15 46
15 29
4 29
14 29
10 35
10 49
15 18
8 10
19 29
0 10
8 32
8 38
35 44
29 41
30 32
27 38
29 31
15 28
9 14
7 32
21 29
1 19
19 23
18 33
6 27
4 40
27 34
31 47
4 17
11 21
3 41
22 32
39 41
20 46
14 26
27 37
16 17
27 42
4 36
12 17
7 25
8 43
9 13
44 48
2 15
2 45
5 38
17 24
