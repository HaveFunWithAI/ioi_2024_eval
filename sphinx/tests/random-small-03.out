50 404
19 16 26 11 46 36 4 14 33 26 34 15 0 7 16 35 24 19 21 26 35 12 34 35 16 20 17 19 41 14 3 19 36 22 34 13 11 47 17 12 47 11 37 46 12 21 25 4 33 37
